pricegraph/src/orderbook.rs

```diff
@@ -57,7 +57,10 @@ impl Orderbook {
         let mut orders = OrderCollector::default();
         let mut users = UserMap::default();
 
-        for element in elements.into_iter().filter(|e| !is_dust_order(e)) {
+        for element in elements
+            .into_iter()
+            .filter(|e| should_include_auction_element(e))
```

Comment:
I predict clippy will complain that this should be

```suggestion
            .filter(should_include_auction_element)
```

Reply:
:see_no_evil: Nice catch.